### PR TITLE
Better explanation of the combat mission due date

### LINKED ISSUE #5890: Combat mission can fail even though the status shows "Completed"
Not a bug, maybe a feature request?

### Observed behaviour

In Combat mission where you have to fight some ruffian, after they are eliminated the mission status shows "Complete" even though I have not claimed my payment yet
image

However, if I fail to claim the payment either by entering the specified system or docking to a valid station before the due date the mission has failed!

### Expected behaviour

Once a mission status is Complete, it should not fail. Maybe another status should be introduced? Like "Payment awaiting" or something like that so the player understands the clock is still ticking :slightly_smiling_face: 
Or still "Active" but the mission wording states clearly that the payment needs to be claimed before the due date

### Steps to reproduce

Take any Combat mission
Load the saved game
=> The mission status is "Completed"
Wait for the due date to expire
Go claim the payment
=> The mission has failed

__My pioneer version (and OS):__
20240710 (Linux)

__My output.txt (required) and game save (optional, but recommended)__
[output.txt](https://github.com/user-attachments/files/16652791/output.txt)
[mission completed.gz](https://github.com/user-attachments/files/16652792/mission.completed.gz)

### PR DESCRIPTION
**Pull request by robothauler**
Proposal to close #5890
The texts are shown when "How will I be paid?" is clicked.

### CODE REVIEW

File: data/lang/module-combat/en.json [hunk 1]
```diff
@@ -189,7 +189,7 @@
   },
   "HARVESTING_LAND_THERE": {
     "description": "",
-    "message": "We have a salesperson in every nearby star system. This person will get in contact with you when landed."
+    "message": "We have a salesperson in every nearby star system. This person will get in contact with you when landed. Please note that we will not pay you if we receive the result of the mission after the due date!"
```
Comment (impaktor) [inline data/lang/module-combat/en.json:192]: Sounds like it's addressing the issue a bit too much.
hand-wavey suggestion:
"...when landed for payment before due date".

File: data/lang/module-combat/en.json [hunk 2]
```diff
@@ -205,7 +205,7 @@
   },
   "MEET_ME_THERE": {
     "description": "",
-    "message": "Meet me in the {rendezvous} system to finish the job!"
+    "message": "Meet me in the {rendezvous} system before the due date. Otherwise I will not pay you!"
```
Comment (impaktor) [inline data/lang/module-combat/en.json:208]: I'd just drop that last sentence. Sounds off, some how.

File: data/lang/module-combat/en.json [hunk 3]
```diff
@@ -225,7 +225,7 @@
   },
   "MILITARY_LAND_THERE": {
     "description": "",
-    "message": "Visit a station in the territory protected by the {org}!"
+    "message": "Visit a station in the territory protected by the {org}! This must be done before the due date! Otherwise the entire operation will fail."
```
Comment (impaktor) [inline data/lang/module-combat/en.json:228]: Combine the sentences to one, perhaps something like: "The operation requires targets be eliminated by date" 
(assuming my guesstimation of mission flavor fits)

**Review by impaktor — COMMENTED** [review]
It's a bit too "on the nose", in my opinion, thus I've made some suggested re-wordings. Feel free to re-write them as you see fit. Just feels like some are _really_ addressing the issue.

**Review by Web-eWorks — APPROVED** [review]

**Pull request merged**